src/ios/CDVKeyboard.m

```diff
@@ -227,6 +244,9 @@ - (void)shrinkView:(CDVInvokedUrlCommand*)command
         }
 
         self.shrinkView = [value boolValue];
+        // Scroll webview content to bottom
+        CGPoint bottomOffset = CGPointMake(0.0f, 0.0f);
+        [self.webView.scrollView setContentOffset:bottomOffset animated:NO];
```

Comment:
This function `shrinkView` is only meant to set the variable above. Should not make changes to the scrollView.